### DISCUSSION TITLE
How to get bdd100k data?

### DISCUSSION THREAD

**Discussion by gly2000** [ethvis/qd-track#16] [Q&A] [answered]
We can not register in bdd-data.berkeley.edu, can you give us one way to download the data. Thank you.

**Comment by fyu** [comment, marked as answer]
Why can't you register?

**Reply by gly2000** [reply]
Thank you, it is right now. The error message "Password must contains at least one digit
at least one lowercase character" is missing, so the "register" button is no response.

**Reply by fyu** [reply]
The data downloading website should be working well now. If you have further questions, please ask in bdd100k discussion: https://github.com/bdd100k/bdd100k/discussions

**Reply by QiyuLuo** [reply]
> 
Now the data set of this website has failed to download, could you please provide the link to the web disk?
